tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/UpToDate/BuildUpToDateCheckTests.cs

```diff
@@ -1342,7 +1342,7 @@ public async Task IsUpToDateAsync_True_Sets_OutputOnly()
                 """);
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/project-system/issues/8642")]
```

Comment:
This (and other skips) are not needed with https://github.com/dotnet/project-system/pull/8645.

Reply:
That PR has merged, so you can rebase/merge and verify they work again.